packages/frontend/app/root.tsx

```diff
@@ -183,7 +206,9 @@ export function ErrorBoundary() {
 
   let errorMessage = 'Unknown error'
   if (error instanceof Error) {
-    errorMessage = error.message
+    errorMessage = error.message.includes('401')
+      ? 'Unauthorized. Please set your API credentials.'
+      : error.message
```

Comment:
On 401 (missing or bad credentials), shows our error page with this message and prompt to return to homepage where credentials can be set. 
![image](https://github.com/user-attachments/assets/90f2e193-0f4c-4a3f-8d67-134a47c29951)